students/scottebetts/lesson02/Fizz-Buzz.py

```diff
@@ -0,0 +1,13 @@
+def fizzbuzz(n):
+    if n % 3 and n % 5 == 0:
+        print('Fizzbuzz')
+    elif n % 3 == 0:
+        print('Fizz')
+    elif n % 5 == 0:
+        print('Buzz')
+    else:
+        print(n)
+
+for i in range(1, 100):
+    fizzbuzz(i)
```

Comment:
You'd actually want to put this for loop inside your fizzbuzz function.  Something like:
`def fizzbuzz(n):
    for i i range (1,n):
        if i % 3 == 0`
And so on.